### PR TITLE
LPD-30984 fix clustering functional test that using 8080 in User.loginPG

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina,

As talked, in the clustering functional test, we will use 8080 for User.loginPG all the time even though we set nodeport 9080.
Take `Clustering#CanAccessGogoShellOnAnyClusteredNode as example`
Before the fix: 
Screenshot_20240708_154953

[Please see here](https://ff2052afe78c5eb8b59787749afbf37c3d66f62d3d9b86282aa73c6-apidata.googleusercontent.com/download/storage/v1/b/testray-results/o/2024-07%2Ftest-1-26%2Ftest-portal-acceptance-pullrequest(master)%2F13682%2Ffunctional-tomcat90-mysql57-jdk8%2F0%2F1%2Fjenkins-console.txt.gz?jk=AYBlUPCH5z8D1Tdr4wO60i06VFEV7A93c8PBmWu-fPo7TVfZPHG7AIJvOURr1SqZT1On1qlt-innNICqWfyCGvOvc1pwRVrMfYK22EDOBZKZMxDpu7Nws34Zl0XwLAO5D9oVdPvKdKazareDuUtWvWpp8ILRjEQsaRLHQrWdfizFmDrxhQkmOVgKif7CSQs3rpEIczUNodZyAjSJgRyYl-VDNZVrP6YNGui54QZaQL_LqENQcaCZu_fiKAV_XreTh8l_c1Bw6HOXoS__J4etNG53lEn2GUYuajzh5ZiV1piWTDki_thQuVvjt4jZmNImgCOzj_1HYlc_w-zwDF7s8QcXgljj870pUdZtU7uG7BJDiUWWeCu_JnnYjzQ79M3Tl74cMr4ZO68UHY6py1qIEd0DFq1yybwyLKyBdCkli6wuq_6_-1xMZtYFFOpm633JSIOhDVaIJxIXJBQqMGakEGH6IiAwZDDNFYIhGKuQXjL7xvJNyVAt7FHCuysxlyyMbxcju7SI2ZR9Gl-H2pyMy2HQw6aB7mf0A3XUsO4etLjKRIz0pm3qGUG5ujyWU9gSxKxPeQdJhsYwVTjE4aNXMDDEfqv0XVZ1IIuECA8prQkmxDff9NxbmndnjcQpu68yOG4GiNrpym3vUfJSQ72_9mNTSPwoPNfIMdDVvb-HV8L910peJCdHO4doBLPoE1fiBXR2YK2kel6PS-5DAHYhU7EHkdS2W-efwIlsRutEdkdRoDczz_VvawylInqFQ-YgBY6HRfb8WDM13NIWCIZCjC8xKdBpBjJ04qHJI8hQ2COKnFpTAlDHj6fx272pphgFwd7D_UWgaLEBX2me0b45KtmZtGyzfCUlQBQElug3f88CnTOdUboVwJ0YdnACTnEEBVIWU76agGIzT8bl1_or_53HofvdMX_HJKTs0s4zIYmxL-BRflxIxHM3jPxkmUURk3vinfCADWcPycq4CJDK9wDwT_gbmP7vQ_D_40iLXjpEYaN6s0If906HNCqjcP02OJ-VQxVg-W4AzeQr5MN-yRsX-m21IQJIPoc25LVTt1HmIrqgh7kJOjgIuAa42W9nSAfQax1cPe8utlc2IgSNplPzrNEb0lyBJpHz3yUpKDpm-bKWe4hE9zxL11HStyBZyxZi3ZzfEaK2P2w2fnymhbzBV7vjhsuejSGNhNhuRK-ABbaPYQj5ivOE77tXFq-ZVb6ZGYs5hG5OWazNong9hnx2kKaGAOn_A0CvfJoRQba1MT9Z601ENPginaT7kg8PIVA5ek7ujTxSmD6fbbbA9dvWBugGIn_-c95AC5aZKCE4xog-PmwzJjJXyOB5mfsZNCyyjF8&isca=1)

After the fix:
Screenshot_20240708_155020
[Please see here](https://ffb28b67e6955e67c4f650bad727d9244fc6dd8a30b3a9604d232e1-apidata.googleusercontent.com/download/storage/v1/b/testray-results/o/2024-07%2Ftest-1-38%2Ftest-portal-acceptance-pullrequest(master)%2F10904%2Ffunctional-tomcat90-mysql57-jdk8%2F0%2F4%2Fjenkins-console.txt.gz?jk=AYBlUPD0SAfi-9ZzE_FsEubZ_OxbZePpAHWKn6_EO97UpAvfOCb5T8OkS4AenEAw61giY1-Xbf7F0LYUwKNGpclXcjd-5CJ7KpPcP9I6E_-1flTWbxRRf6MfvIYv9GvCHXinu453WmsGBv18LIU5gfJHqURz79kJicBXpAVvQX0mynDOUOlYYBLZ9ocJwcLZIXUH7SbeaCq80zj6td-nsnB19R15IsKj2GUYa0rnqX4dUjSPmafrllWddxWQvsp-03CVrTInkrkn9GQpGY38LjOPxghY-iqhB7fCOfhgVLMVxCHR_CrAr5WzbmN8rt7iM_Up96di6LZhofMgZpzyL2W0YAWJrrTa4me3hv8j9EoZ6jZLrKqJb3H-HQ3lxn75JKf89k-pmavQ0yJFYSQkxr6uMhIB8dpwV0MVxFa3uvPHKYLCyAs-4e3LgdbyCW0vpjXUlsfjZHo-RqkRKaNPnwAtj6KG683NhYRAgaWj0WhwzH-iISG6QRXObXyCHwyMa6NN71ISTo3sc_EYOck_zUuDMIDfHngywx2v-wvLrdn5nL4JeHuRehM88NjrE38znUHjFN7ZbG7DXdpfKUzkFBJQ9etO88ObpJYi86gXU5ugzTu8DNI537BSwhGfD6gaAzDwPyEuFKzn8P8n2qMf2Y7YzYD7avRd1Bi7Dh7JBCZ2JhpKb_UWaMPCYoO4BaRqvIHFx4bHaoCuhyjCcViS1MN00i6k4foH45K-DYrfNxY1Jp1zliJWmMQoWbGJEtxaD4A5sLnHkIRxOa6FeSrHi0Uh-Gl-DE2QDnZN3GXclh9tuBeoMaRMCvHWQzFPmSDok6VcCuOvtP28g71C6ZsEwMOI2bMAK0gwayBBAGQHYRRzoUZcR0uwZH-H1iN4JgbmqS813NvvY6EYaZjHowEzYK7mk57q6wigeIzOWGBh7yuPrk1Cxi1YJCjY4tp080yd26zTmUlXpObVjDfBZh6S_QgHR-o9u9PRxNGzhSxO6eTvuvmSuQEipf_GcFtaCLg8f8ZJve9cN-LnxIAxNO13m2PBNN9MUmm77kuzlogXflp-KJnk5S6Gr3B90gtBQFZJfOVIQYqzT-g2eXLOv8KW7u_kYL5jK3FLy7sbByPIgl2MVDYEy_WyeGpmmp8kI6saQ2lKvbeq71QvzMBpkXjvBoMct0WMP1mEvbzWNebec9Vk1ENGsM-dLjSQ5LmuYfSgR_4f-nJHJK_brK-JUZPsiq05ECJU5zjdGXUVRmzEeTHy77MhCCIuDuN-kjjVhLfQmjnkEEHb5d3351kEDkYr17vP5eLjBV00BpetLxPdq_e17j6X63rXAujo_ChTQOaNJPCRhq4&isca=1)

cc: @julschong

### CODE REVIEW

**Comment by jeffwu0724** [comment]
ci:test:clustering

**Comment by jeffwu0724** [comment]
ci:test:relevant

**Comment by jeffwu0724** [comment]
ci:stop

**Comment by jeffwu0724** [comment]
forced push to use the sub-task ticket number LPD-30984

**Comment by jeffwu0724** [comment]
ci:test:clustering

**Comment by jeffwu0724** [comment]
ci:test:sf

**Comment by jeffwu0724** [comment]
ci:test:clustering

File: portal-web/test/functional/com/liferay/portalweb/macros/json/company/JSONCompany.macro [hunk 1]
```diff
@@ -119,12 +119,15 @@ definition {
...
 	@summary = "Default summary"
-	macro getCompanyIdNoSelenium(token = null) {
+	macro getCompanyIdNoSelenium(token = null, portalURL = null) {
 		if (!(isSet(portalInstanceName))) {
 			var portalInstanceName = JSONCompany.getPortalInstanceNameNoSelenium();
 		}
 
-		if (!(isSet(portalURL))) {
+		if (isSet(portalURL)) {
+			var portalURL = ${specificURL};
```
Comment (julschong) [inline portal-web/test/functional/com/liferay/portalweb/macros/json/company/JSONCompany.macro:128]: @jeffwu0724 hello

File: portal-web/test/functional/com/liferay/portalweb/macros/json/company/JSONCompany.macro [hunk 2]
```diff
@@ -119,12 +119,15 @@ definition {
...
+		}
+		else {
 			var portalURL = JSONCompany.getDefaultPortalURL();
```
Comment (julschong) [inline portal-web/test/functional/com/liferay/portalweb/macros/json/company/JSONCompany.macro:131]: remove the var

**Comment by jeffwu0724** [comment]
ci:test:clustering

**Comment by julschong** [comment]
ci:report:34528405

**Comment by jeffwu0724** [comment]
ci:report:35191325

**Comment by jeffwu0724** [comment]
ci:report:35191325

**Comment by jeffwu0724** [comment]
Found unique regression, looking into it...
https://testray.liferay.com/#/project/34754/routines/34756/build/35191325/case-result/35210321/history
https://testray.liferay.com/#/project/34754/routines/34756/build/35191325/case-result/35210461/history
https://testray.liferay.com/#/project/34754/routines/34756/build/35191325/case-result/35210471/history

**Pull request closed without merging**